packages/global-cli/README.md

```diff
@@ -1,126 +1,65 @@
+It will create a global `react-native` binary (even though the package name is `react-native-cli`).
 
-Now you can run sinopia by simply doing:
+## Usage
 
-    $ sinopia
+To create a new React Native Project called "AwesomeProject" you can run:
 
-Running it for the first time creates a default config file. Open `~/.config/sinopia/config.yaml` and configure it like this (note the `max_body_size`):
+```sh
+react-native init AwesomeProject
+```
 
-    storage: ./storage
+It will install `react-native`, `react`, `jest` and a bunch of other necessary packages from the [default template](https://github.com/facebook/react-native/tree/master/template).
 
-    auth:
-      htpasswd:
-        file: ./htpasswd
+After it's finished, your AwesomeProject should be ready to use. From this point, you should use your local `react-native` binary to run the proper [React Native CLI](../cli):
 
-    uplinks:
-      npmjs:
-        url: https://registry.npmjs.org/
+```sh
+yarn react-native link native-dep
+```
 
-    packages:
-      'react-native':
-        allow_access: $all
-        allow_publish: $all
+## Options
 
-      'react-native-cli':
-        allow_access: $all
-        allow_publish: $all
+### `--template`
 
-      '**':
-        allow_access: $all
-        proxy: npmjs
+Use a custom template.
 
-      '*':
-        allow_access: $all
-        proxy: npmjs
+Example: this will install init your AwesomeProject using template called `react-native-template-samplename` from npm:
 
-    logs:
-      - {type: stdout, format: pretty, level: http}
+```sh
+react-native init AwesomeProject --template samplename
+```
 
-    max_body_size: '50mb'
+You can also pass remote git address and local filepath as a `--template` parameter.
 
-Remember to restart sinopia afterwards.
+### `--version`
 
-### Publishing to sinopia
+Use a custom version. By default `react-native init` will use the latest stable.
 
-Now we need to publish the two React Native packages to our local registry. To do this, we configure npm to use the new registry, unpublish any existing packages and then publish the new ones:
+Example: this will install init your AwesomeProject using version `0.57.0`:
 
-    react-native$ npm set registry http://localhost:4873/
-    react-native$ npm adduser --registry http://localhost:4873/
-    # Check that it worked:
-    react-native$ npm config list
-    react-native$ npm unpublish --force
-    react-native$ npm publish
-    react-native$ cd react-native-cli/
-    react-native-cli$ npm unpublish --force
-    react-native-cli$ npm publish
+```sh
+react-native init AwesomeProject --version 0.57.0
+```
 
-### Running the local CLI
+You can also install a specific tag, like `next`, using:
 
-Now that the packages are installed in sinopia, you can install the new `react-native-cli` package globally and when you use `react-native init`, it will install the new `react-native` package as well:
+```sh
+react-native init AwesomeProject --version react-native@next
```

Comment:
@grabbou mind reviewing this section? It's not clear to me when users should use `react-native@0.57.0` and just `0.57.0`